test/bats/user-login.bats

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bats
+
+load "helpers/setup-and-teardown"
+
+setup_file() {
+  clear_cache
+
+  start_server
+}
+
+teardown_file() {
+  stop_server
+}
+
+@test "user-login: login user" {
```

Comment:
shouldn't this be a helper? (or at least part of it)

Reply:
We already have a helper for it. This is to explicitly test it outside the helper in an isolated way.

Not sure if this approach is repetitive/redundant? I like that we're explicitly asserting this as a test condition.

Reply:
I think is not necessary.. if login is not working the other tests will fail but if you want to be explicit you can include tests for no happy paths

Reply:
Agree - let's not add this. It just increases the e2e runtime but doesn't assert anything unique.